src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -143,16 +150,48 @@ function DisplayLine({
   )
 }
 
+function DisplayRulingDollarLine({
+  line,
+  container = 'div'
+}: {
+  line: Line
+  container?: string
+}): JSX.Element {
+  const rulingDollarLine = line as RulingDollarLine
+  const element = `Transliteration__${rulingDollarLine.type}`
+  const number = capitalizeFirstLetter(
+    rulingDollarLine.display_value.split(' ')[0]
```

Comment:
Use the correct property from the line model (`number`) instead of adding hacks.

Maybe adding the display value is not a good idea after all?